src/dashboard/pages/create_graph/create_graph.py

```diff
@@ -3,30 +3,42 @@
 This module displays different types of graphs based on input,
 from either a csv-file or from a database.
 """
+
 import base64
 import io
+from typing import Any
 
 import dash
-from dash import callback, dcc, html
-from dash.dependencies import Component, Input, Output, State
+from dash import Patch, State, callback, dcc, html
+from dash.dependencies import Component, Input, Output
 from dash.exceptions import PreventUpdate
-import plotly.express as px
+import dash_bootstrap_components as dbc
+import plotly.graph_objs as go
 import polars as pl
 
 from dashboard.components import button, icon
 
 dash.register_page(__name__, path="/create-graph", nav_item=False)
 
+# used to supress warning messages for all components created by layout
+suppress_callback_exceptions = True
+
+# for debugging
+global debug
```

Comment:
`global` statements are only needed inside a functions when writing to a globla variable. Here it does nothing.

Reply:
Removed